[Dev] CI with pyarrow failed

### What happens?

Some Python CI workflows (including [this](https://github.com/duckdb/duckdb/actions/runs/4025058391/jobs/6917795032#step:7:524)) failed with the following error.

```
  =================================== FAILURES ===================================
  _______________ TestArrowRecordBatchReader.test_parallel_reader ________________
  
  self = <test_arrow_recordbatchreader.TestArrowRecordBatchReader object at 0x7f32006b6910>
  duckdb_cursor = <duckdb.DuckDBPyConnection object at 0x7f3207c1e130>
  
      def test_parallel_reader(self,duckdb_cursor):
          if not can_run:
              return
      
          duckdb_conn = duckdb.connect()
          duckdb_conn.execute("PRAGMA threads=4")
      
          parquet_filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'data','userdata1.parquet')
      
          userdata_parquet_dataset= pyarrow.dataset.dataset([
              parquet_filename,
              parquet_filename,
              parquet_filename,
          ]
          , format="parquet")
      
          batches= [r for r in userdata_parquet_dataset.to_batches()]
  >       reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
  
  /project/tests/fast/arrow/test_arrow_recordbatchreader.py:31: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  >   ???
  E   TypeError: from_batches() takes exactly 1 positional argument (2 given)
  
  pyarrow/_dataset.pyx:2686: TypeError
  ______ TestArrowRecordBatchReader.test_parallel_reader_replacement_scans _______
  
  self = <test_arrow_recordbatchreader.TestArrowRecordBatchReader object at 0x7f32005c6ca0>
  duckdb_cursor = <duckdb.DuckDBPyConnection object at 0x7f3207c1e130>
  
      def test_parallel_reader_replacement_scans(self,duckdb_cursor):
          if not can_run:
              return
      
          duckdb_conn = duckdb.connect()
          duckdb_conn.execute("PRAGMA threads=4")
      
          parquet_filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'data','userdata1.parquet')
      
          userdata_parquet_dataset= pyarrow.dataset.dataset([
              parquet_filename,
              parquet_filename,
              parquet_filename,
          ]
          , format="parquet")
      
          batches= [r for r in userdata_parquet_dataset.to_batches()]
  >       reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
  
  /project/tests/fast/arrow/test_arrow_recordbatchreader.py:56: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  >   ???
  E   TypeError: from_batches() takes exactly 1 positional argument (2 given)
  
  pyarrow/_dataset.pyx:2686: TypeError
  ___________ TestArrowRecordBatchReader.test_parallel_reader_register ___________
  
  self = <test_arrow_recordbatchreader.TestArrowRecordBatchReader object at 0x7f32006b6f70>
  duckdb_cursor = <duckdb.DuckDBPyConnection object at 0x7f3207c1e130>
  
      def test_parallel_reader_register(self,duckdb_cursor):
          if not can_run:
              return
      
          duckdb_conn = duckdb.connect()
          duckdb_conn.execute("PRAGMA threads=4")
      
          parquet_filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'data','userdata1.parquet')
      
          userdata_parquet_dataset= pyarrow.dataset.dataset([
              parquet_filename,
              parquet_filename,
              parquet_filename,
          ]
          , format="parquet")
      
          batches= [r for r in userdata_parquet_dataset.to_batches()]
  >       reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
  
  /project/tests/fast/arrow/test_arrow_recordbatchreader.py:78: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  >   ???
  E   TypeError: from_batches() takes exactly 1 positional argument (2 given)
  
  pyarrow/_dataset.pyx:2686: TypeError
  _________ TestArrowRecordBatchReader.test_parallel_reader_default_conn _________
  
  self = <test_arrow_recordbatchreader.TestArrowRecordBatchReader object at 0x7f32005c6280>
  duckdb_cursor = <duckdb.DuckDBPyConnection object at 0x7f3207c1e130>
  
      def test_parallel_reader_default_conn(self,duckdb_cursor):
          if not can_run:
              return
      
          parquet_filename = os.path.join(os.path.dirname(os.path.realpath(__file__)),'data','userdata1.parquet')
      
          userdata_parquet_dataset= pyarrow.dataset.dataset([
              parquet_filename,
              parquet_filename,
              parquet_filename,
          ]
          , format="parquet")
      
          batches= [r for r in userdata_parquet_dataset.to_batches()]
  >       reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
  
  /project/tests/fast/arrow/test_arrow_recordbatchreader.py:99: 
  _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  
  >   ???
  E   TypeError: from_batches() takes exactly 1 positional argument (2 given)
  
  pyarrow/_dataset.pyx:2686: TypeError
  =========================== short test summary info ============================
  FAILED ../../../project/tests/fast/arrow/test_arrow_recordbatchreader.py::TestArrowRecordBatchReader::test_parallel_reader
  FAILED ../../../project/tests/fast/arrow/test_arrow_recordbatchreader.py::TestArrowRecordBatchReader::test_parallel_reader_replacement_scans
  FAILED ../../../project/tests/fast/arrow/test_arrow_recordbatchreader.py::TestArrowRecordBatchReader::test_parallel_reader_register
  FAILED ../../../project/tests/fast/arrow/test_arrow_recordbatchreader.py::TestArrowRecordBatchReader::test_parallel_reader_default_conn
  ============= 4 failed, 600 passed, 1 skipped in 229.83s (0:03:49) =============
```

### To Reproduce

Run Python CI

### OS:

Ubuntu 20.04.5, x64

### DuckDB Version:

master

### DuckDB Client:

Python

### Full Name:

Koki Ueha

### Affiliation:

None

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

--- a/tools/pythonpkg/tests/fast/arrow/test_arrow_recordbatchreader.py
+++ b/tools/pythonpkg/tests/fast/arrow/test_arrow_recordbatchreader.py
@@ -28,7 +28,7 @@ class TestArrowRecordBatchReader(object):
         , format="parquet")
 
         batches= [r for r in userdata_parquet_dataset.to_batches()]
-        reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
+        reader=pyarrow.dataset.Scanner.from_batches(batches,schema=userdata_parquet_dataset.schema).to_reader()
 
         rel = duckdb_conn.from_arrow(reader)
 
@@ -53,7 +53,7 @@ class TestArrowRecordBatchReader(object):
         , format="parquet")
 
         batches= [r for r in userdata_parquet_dataset.to_batches()]
-        reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
+        reader=pyarrow.dataset.Scanner.from_batches(batches,schema=userdata_parquet_dataset.schema).to_reader()
 
         assert duckdb_conn.execute("select count(*) from reader where first_name=\'Jose\' and salary > 134708.82").fetchone()[0] == 12
         assert duckdb_conn.execute("select count(*) from reader where first_name=\'Jose\' and salary > 134708.82").fetchone()[0] == 0
@@ -75,7 +75,7 @@ class TestArrowRecordBatchReader(object):
         , format="parquet")
 
         batches= [r for r in userdata_parquet_dataset.to_batches()]
-        reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
+        reader=pyarrow.dataset.Scanner.from_batches(batches,schema=userdata_parquet_dataset.schema).to_reader()
 
         duckdb_conn.register("bla", reader)
 
@@ -96,7 +96,7 @@ class TestArrowRecordBatchReader(object):
         , format="parquet")
 
         batches= [r for r in userdata_parquet_dataset.to_batches()]
-        reader=pyarrow.dataset.Scanner.from_batches(batches,userdata_parquet_dataset.schema).to_reader()
+        reader=pyarrow.dataset.Scanner.from_batches(batches,schema=userdata_parquet_dataset.schema).to_reader()
 
         rel = duckdb.from_arrow(reader)
 
@@ -104,4 +104,3 @@ class TestArrowRecordBatchReader(object):
         # The reader is already consumed so this should be 0
         assert rel.filter("first_name=\'Jose\' and salary > 134708.82").aggregate('count(*)').execute().fetchone()[0] == 0
 
-    

